xonsh/xontribs_meta.py

```diff
@@ -0,0 +1,639 @@
+import functools
+import typing as tp
+
+
+class XontribPkg(tp.NamedTuple):
+    install: tp.Dict[str, str]
+    license: str = ""
+    name: str = ""
+    url: tp.Optional[str] = None
+
+
+class Xontrib(tp.NamedTuple):
```

Comment:
This relationship seems flipped to me.  A XontribPkg seems like it would hold many Xontribs

Reply:
Oh never mind.  I guess I just expected that there would be a separate package dictionary somewhere, since the XontribPkg objects should be unique too. You kind of have that with the "core_pkg"

Reply:
Yes the Xontribs holds one package info.